root/backend/server/api/routes/poll.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express');
+const Poll = require('../../../db/models/poll');
+
+const router = express.Router();
+
+router.route('/')
+  .get(async (req, res, next) => {
+    try {
+      const polls = await Poll.find({}).then(data => {
+        return data.map(poll => {
+          return {
+            id: poll._id.toString(),
+            title: poll.name,
+            location: poll.location
+          }
+        });
+      });
+      res.json({ response: polls });
+      res.end();
```

Comment:
do you _need_ `res.end()` here? can you instead just send a status and the data like so?

```suggestion
      res.status(200).json({ response: polls });
```